### DISCUSSION TITLE
How to read the Temp basal delta value from Loopalyzer Nightscout report?

### DISCUSSION THREAD

**Discussion by janvitos** [NightscoutFoundation/xDrip#3496] [General]
Hello,

I am trying to figure out how much more basal insulin the loop is giving my son at different times of the day by analyzing the Temp basal delta values from the Loopalyzer Nightscout report.

My son's regular basal is 0.5 U/h. His Temp basal delta is often 0.5. How do I use the 0.5 in relation with the 0.5 U/h? Do I simply do 0.5 * 1.5 = 0.75, which would equal to a delta of 0.25? If not, what's the proper math here?

I did not see this mentioned anywhere in the Nightscout documentation. Might be a good idea to add it?

Thanks!

**Comment by Navid200** [comment]
We created a platform to install and run Nightscout on when Heroku stopped to be free.
But, Nightscout itself is not supported here.

I am aware of a facebook group that provides support: https://www.facebook.com/groups/cgminthecloud
There may be other platforms I am not aware of.

**Reply by janvitos** [reply]
> Nightscout itself is not supported here

I had a feeling Nightscout was not supported here, but I didn't know where else to ask, and I thought maybe someone here might've had the answer since you are all such resourceful people! Unfortunately, I don't use Facebook, so I can't ask my question there.

Do you mind if I keep this thread open in case someone knows the answer? Hope it's not too much of a problem.

Thanks!

**Reply by Navid200** [reply]
Any subject related to xDrip can be discussed here.
The discussions can be kept open.  There is no benefit in closing a discussion unless you want it to become quiet.
So, yes, please keep this open.

This is the Nightscout repository:
https://github.com/nightscout/cgm-remote-monitor
You can post in their issues and ask considering they have not opened their discussions.  
They may give you directions to where else you can ask questions.

**Reply by janvitos** [reply]
> You can post in their issues and ask considering they have not opened their discussions.
> They may give you directions to where else you can ask questions.

Good idea. Thanks!
